### PR TITLE
Add skopeo to analysis-runner driver image

### PR DESCRIPTION
**Pull request by lgruen**
This is handy for copying Docker images into the Artifact Registry.

Context: https://centrepopgen.slack.com/archives/C018KFBCR1C/p1641942379022300?thread_ts=1641940218.018900&cid=C018KFBCR1C

### CODE REVIEW

File: driver/Dockerfile [hunk 1]
```diff
@@ -14,8 +14,19 @@ RUN apt-get update && apt-get install -y git wget bash bzip2 zip && \
...
+        cpg-utils \
+        phantomjs \
+        r-argparser && \
+        r-base=4.1.1 \
+        r-essentials \
+        r-tidyverse \
+        sample-metadata=3.5.0 \
+        selenium \
+        skopeo \
+        statsmodels \
```
Comment (MattWellie) [inline driver/Dockerfile:29]:
```suggestion
        statsmodels; \
```

Comment (MattWellie) [reply driver/Dockerfile:29]: final import no longer followed by && or a termimation

**Review by MattWellie — CHANGES_REQUESTED** [review]
Alphabetic and spaced out, I've been itching to make that change

File: driver/Dockerfile [hunk 2]
```diff
@@ -14,8 +14,19 @@ RUN apt-get update && apt-get install -y git wget bash bzip2 zip && \
...
     micromamba install -y --prefix $MAMBA_ROOT_PREFIX \
         -c cpg -c bioconda -c conda-forge \
-        hail=$HAIL_VERSION cpg-utils analysis-runner sample-metadata=3.5.0 bokeh selenium phantomjs statsmodels \
-        r-base=4.1.1 r-essentials r-tidyverse r-argparser && \
+        hail=$HAIL_VERSION \
+        analysis-runner \
+        bokeh \
+        cpg-utils \
+        phantomjs \
+        r-argparser && \
```
Comment (MattWellie) [inline driver/Dockerfile:22]:
```suggestion
        r-argparser \
```

File: driver/Dockerfile (hunk 1 above)
Comment (lgruen) [reply driver/Dockerfile:29]: Thanks a lot for catching this! I've changed it to `&&` to make sure we stop immediately in case of an error.

**Review by MattWellie — APPROVED** [review]
LGTM

**Pull request merged**